modules/manage/partials/tiered-storage.adoc

```diff
@@ -1329,6 +1329,47 @@ ifndef::env-kubernetes[Edit them with the `rpk cluster config edit` command.]
 | The public certificate used to validate the TLS connection to object storage. If this is empty, Redpanda uses your operating system's CA cert pool.
 |===
 
+== Faster partition movement during cluster resize
+
+Tiered Storage gives you the option to boost node operations particularly when resizing a cluster. For instance, adding a new node to increase capacity for an overburdened cluster could introduce additional stress as partition replicas are transferred to the new node. For cloud deployments, commissioning or decommissioning a broker can be a slow and expensive process, especially across multiple availability zones (AZ).
+
+Tiered Storage can help make the cluster resize process more cost-efficient by leveraging data that has already been uploaded to object storage. Instead of transferring all data from local storage to the reassigned replicas, the replicas can instead be initialized using object storage.
```

Comment:
'faster and more cost efficient'

Reply:
@kbatuigas 'initialized from cloud storage' might be misleading as we down download data from cloud storage currently.   

'the replicas can instead be initialized to rely more heavily on Tiered Storage for read requests.